server/routes/file/read.js

```diff
@@ -529,6 +509,8 @@ router.get('/downloadSchema', [
 
 });
 
+
+// NOT IN USE
```

Comment:
We not currently using this route, I have not remove it for ref. After we changed Graph we no longe allow user to connect file to file so we do not use license inheritance, maybe will be revisited later, currently not in use